test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+Año Nuevo  #!#Año Nuevo  #!#Año Nuevo#!#Año Nuevo#!#Año Nuevo#!#Año Nuevo#!#Año Nuevo#!#Año Nuevo
+árbol      #!#árbol      #!#árbol#!#árbol#!#árbol#!#árbol#!#árbol#!#árbol
+café       #!#café       #!#café#!#café#!#café#!#café#!#café#!#café
+canapé     #!#canapé     #!#canapé#!#canapé#!#canapé#!#canapé#!#canapé#!#canapé
+chaptéR    #!#chaptéR    #!#chaptéR#!#chaptéR#!#chaptéR#!#chaptéR#!#chaptéR#!#chaptéR
+jalapeño   #!#jalapeño   #!#jalapeño#!#jalapeño#!#jalapeño#!#jalapeño#!#jalapeño#!#jalapeño
+naïve      #!#naïve      #!#naïve#!#naïve#!#naïve#!#naïve#!#naïve#!#naïve
+película   #!#película   #!#película#!#película#!#película#!#película#!#película#!#película
+Piñata     #!#Piñata     #!#Piñata#!#Piñata#!#Piñata#!#Piñata#!#Piñata#!#Piñata
+résumé     #!#résumé     #!#résumé#!#résumé#!#résumé#!#résumé#!#résumé#!#résumé
+TELÉFONO   #!#TELÉFONO   #!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO
+TEññi?     #!#TEññiȘ     #!#TEññi?#!#TEññiȘ#!#TEññi?#!#TEññiȘ#!#TEññi?#!#TEññi?
+~~END~~
+
+
+-- Create a new table with employees hired after 2015
+SELECT * INTO test_db_collation_vu_prepare_db121_si2 FROM test_db_collation_vu_prepare_db121_t1 t1 WHERE t1.nv > 'naive' ORDER BY nv;
+GO
+
+SELECT * FROM test_db_collation_vu_prepare_db121_si2 ORDER BY nv;
+GO
+~~START~~
+char#!#nchar#!#varchar#!#nvarchar#!#text#!#ntext#!#sql_variant#!#varchar
+película   #!#película   #!#película#!#película#!#película#!#película#!#película#!#película
+Piñata     #!#Piñata     #!#Piñata#!#Piñata#!#Piñata#!#Piñata#!#Piñata#!#Piñata
+résumé     #!#résumé     #!#résumé#!#résumé#!#résumé#!#résumé#!#résumé#!#résumé
+TELÉFONO   #!#TELÉFONO   #!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO#!#TELÉFONO
+TEññi?     #!#TEññiȘ     #!#TEññi?#!#TEññiȘ#!#TEññi?#!#TEññiȘ#!#TEññi?#!#TEññi?
+~~END~~
+
+
+
+-- Case 6.1: Validate whether correct collation is being picked up or not by SIMPLE SELECT
+USE test_db_collation_vu_prepare_db121;
+GO
+
+SELECT nv FROM test_db_collation_vu_prepare_db121_t1 where nv IN ('cafe', 'JALAPeno', 'ñáIVE', 'pínata') ORDER BY nv;
+GO
+~~START~~
+nvarchar
+café
+jalapeño
+naïve
+Piñata
+~~END~~
+
+
+SELECT nv FROM test_db_collation_vu_prepare_db121_t1 GROUP BY nv ORDER BY nv;
+GO
+~~START~~
+nvarchar
+Año Nuevo
+árbol
+café
+canapé
+chaptéR
+jalapeño
+naïve
+película
+Piñata
+résumé
+TELÉFONO
+TEññiȘ
+~~END~~
+
+
+CREATE VIEW test_db_collation_vu_prepare_db121_v1 AS SELECT nv FROM test_db_collation_vu_prepare_db121_t1 ORDER BY nv;
+GO
+
+SELECT * FROM test_db_collation_vu_prepare_db121_v1;
+GO
+~~START~~
+nvarchar
+Año Nuevo
+árbol
+café
+canapé
+chaptéR
+jalapeño
+naïve
+película
+Piñata
+résumé
+TELÉFONO
+TEññiȘ
+~~END~~
+
+
+SELECT * FROM test_db_collation_vu_prepare_db121_v1 GROUP BY nv ORDER BY nv;
+GO
+~~START~~
+nvarchar
+Año Nuevo
+árbol
+café
+canapé
+chaptéR
+jalapeño
+naïve
+película
+Piñata
+résumé
+TELÉFONO
+TEññiȘ
+~~END~~
+
+
+
+
+CREATE PROCEDURE test_db_collation_vu_prepare_db121_p1 @param NVARCHAR(50)
```

Comment:
is there a way to vefiy the collation of this param?

Reply:
Did not find any convenient way to do so.